cmd/prober/write.go

```diff
@@ -139,6 +139,23 @@ func fulcioWriteEndpoint(ctx context.Context, priv *ecdsa.PrivateKey) (*x509.Cer
 	return cert[0], nil
 }
 
+func makeRekorRequest(cert *x509.Certificate, priv *ecdsa.PrivateKey, hostPath string) (*http.Response, time.Time, error) {
+	body, err := rekorEntryRequest(cert, priv)
+	t := time.Now()
+	if err != nil {
+		return nil, t, fmt.Errorf("rekor entry: %w", err)
+	}
+	req, err := retryablehttp.NewRequest(http.MethodPost, hostPath, bytes.NewBuffer(body))
+	if err != nil {
+		return nil, t, fmt.Errorf("new request: %w", err)
+	}
+	setHeaders(req, "")
+
+	t = time.Now()
+	resp, err := retryableClient.Do(req)
+	return resp, t, err
```

Comment:
I'd suggest returning the latency calculation `latency := time.Since(t).Milliseconds()` rather than the time object.